### DISCUSSION TITLE
How do you solve date input?

### DISCUSSION THREAD

**Discussion by billjenner** [twbs/bootstrap#39288] [Q&A] [answered]
Bootstrap doesn't support date input. To me collecting a date value seems fairly important. Any plans to add support in the future? How do you solve it today - JQuery-UI?

Looking strictly at the Vue framework, Bootstrap vue provides support:
_https://bootstrap-vue.org/docs/components/form-datepicker

**Comment by billjenner** [comment, marked as answer]
Looks like this works well enough:
      
  `    <input type="date" class="form-control">`

**Reply by Ciccioposticcio** [reply]
Thank you. I was looking around the docs of Bootstrap and was frustrated because i cant find a datepicker!
